docs/system/hardware-metrics.md

```diff
@@ -93,11 +93,11 @@ Examples: physical server, switch or disk array.
 **Description:** A battery in a computer system or an UPS.
 
 | Name                      | Description                                                                   | Units | Instrument Type ([*](/docs/general/metrics.md#instrument-types)) | Value Type | Attribute Key(s)                                                            | Attribute Values                                      |
-| ------------------------- | ----------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | --------------------------------------------------------------------------- | ----------------------------------------------------- |
+|---------------------------| ----------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | --------------------------------------------------------------------------- | ----------------------------------------------------- |
 | `hw.battery.charge`       | Remaining fraction of battery charge                                          | 1     | Gauge                                             | Double     |                                                                             |                                                       |
 | `hw.battery.charge.limit` | Lower limit of battery charge fraction to ensure proper operation             | 1     | Gauge                                             | Double     | `limit_type` (Recommended)                                                  | `critical`, `throttled`, `degraded`                   |
 | `hw.battery.time_left`    | Time left before battery is completely charged or discharged                  | s     | Gauge                                             | Int        | `state` (Conditionally Required, if the battery is charging or discharging) | `charging`, `discharging`                             |
-| `hw.status`               | Operational status: `1` (true) or `0` (false) for each of the possible states |       | UpDownCounter                                     | Int        | `state` (**Required**)                                                      | `ok`, `degraded`, `failed`, `charging`, `discharging` |
+| `hw.battery.status`       | Operational status: `1` (true) or `0` (false) for each of the possible states |       | UpDownCounter                                     | Int        | `state` (**Required**)                                                      | `ok`, `degraded`, `failed`, `charging`, `discharging` |
```

Comment:
No, there is absolutely no need to use namespaces everywhere.

It's much easier to query `hw.status{state="failed"}` to cover all devices, rather than `hw.battery.status{state="failed"} + hw.cpu.status{state="failed"} + hw.memory.status{state="failed"} + ...`

Same goes for all metrics that are common to different classes of hardware devices.

BTW, the very first version of the specification used namespaces heavily, and we simplified it and I think it's great. Let's not go backward and get it more complicated.